### PR TITLE
Update to langpack source files for January release

### PR DESCRIPTION
**Pull request by smartguest**
This PR updates the langpack source files and XLFs for the December release.

### CODE REVIEW

**Review by Charles-Gagnon — APPROVED** [review]
Not a problem to do this early - but make sure you're still sync'ing up to the latest strings when we get to code complete for the actual release.

File: i18n/ads-language-pack-de/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 All notable changes to the "ads-language-pack-de" language pack will be documented in this file.
 
 ## [Released]
+* December 1, 2022 - Release for Azure Data Studio 1.41
```
Comment (Charles-Gagnon) [inline i18n/ads-language-pack-de/CHANGELOG.md:5]: This definitely isn't going to be accurate, so maybe wait to update the changelogs until we actually start doing the release.

**Review by alanrenmsft — APPROVED** [review]

File: i18n/ads-language-pack-de/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 All notable changes to the "ads-language-pack-de" language pack will be documented in this file.
 
 ## [Released]
+* January 18, 2022 - Release for Azure Data Studio 1.41
```
Comment (cheenamalhotra) [inline i18n/ads-language-pack-de/CHANGELOG.md:5]: 2023 :)

**Review by kburtram — APPROVED** [review]
LGTM, should this go into today before we branch for Jan release?

**Comment by smartguest** [comment]
@kburtram Yes, following procedure that I established.

**Pull request merged**